api/all/src/test/java/io/opentelemetry/api/OpenTelemetryTest.java

```diff
@@ -6,7 +6,7 @@
 package io.opentelemetry.api;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
```

Comment:
Is there no equivalent in jupiter? I liked the fact that we always use the jupiter imports.

Reply:
The jupiter equivalent would be using `assertThat()` around the return value of the `asserThrows` which is the exception. But we use asserj for all other assertions, so I think either way is equally consistent.